test/rust-test-util/TestResultReportingClient/Cargo.toml

```diff
@@ -5,6 +5,11 @@ authors = ["Azure IoT Edge Devs"]
 edition = "2018"
 description = "Http client for the TestResultCoordinator test orchestration module"
 
+# when fixed we can remove (https://github.com/Lymia/enumset/issues/17)
```

Comment:
Is there a tracking issue with this link as well? Want to make sure we don't forget to check in on this from time to time :)